config/config_test.go

```diff
@@ -266,14 +267,14 @@ func TestGetDefaultServiceTTL(t *testing.T) {
 }
 
 func TestDurationMarshal(t *testing.T) {
```

Comment:
you could remove this test, because duration marshall already tested in aos common

Reply:
Done